File: __tests__/__fixtures__/output/node/unirest/multipart-file.js
const fs = require("fs");
const unirest = require("unirest");

const req = unirest("POST", "https://httpbin.org/anything");

req.headers({
  "content-type": "multipart/form-data; boundary=---011000010111000001101001"
});

req.multipart([
  {
    "body": fs.createReadStream("__tests__/__fixtures__/files/hello.txt"),
    "content-type": "text/plain"
  }
]);

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

File: src/targets/go/native/fixtures/multipart-file.go
package main

import (
	"fmt"
	"strings"
	"net/http"
	"io"
)

func main() {

	url := "https://httpbin.org/anything"

	payload := strings.NewReader("-----011000010111000001101001\r\nContent-Disposition: form-data; name=\"foo\"; filename=\"src/fixtures/files/hello.txt\"\r\nContent-Type: text/plain\r\n\r\n\r\n-----011000010111000001101001--")

	req, _ := http.NewRequest("POST", url, payload)

	req.Header.Add("content-type", "multipart/form-data; boundary=---011000010111000001101001")

	res, _ := http.DefaultClient.Do(req)

	defer res.Body.Close()
	body, _ := io.ReadAll(res.Body)

	fmt.Println(string(body))

}